openage/convert/opus/opusenc.pyx

```diff
@@ -0,0 +1,406 @@
+# Copyright 2016-2017 the openage authors. See copying.md for legal info.
+
+import time
+from libc.string cimport memcpy, memset
+from cpython.mem cimport PyMem_Malloc, PyMem_Free
+from cpython.bytes cimport PyBytes_FromStringAndSize
+
+from ...log import dbg, spam
+
+from . cimport ogg, opus
```

Comment:
`make cythonize` gives me this warning:
`/home/castilma/.repo/ap/openage/openage/convert/opus/opusenc.pyx: cannot find cimported module '.'`